src/MiniProfiler.Shared/ui/lib/MiniProfiler.ts

```diff
@@ -422,27 +433,29 @@ namespace StackExchange.Profiling {
                           key === 'Started' && typeof value === 'string' && isoDate.exec(value) ? new Date(value) : value;
 
                 mp.fetchStatus[id] = 'Starting fetch';
-                this.jq.ajax({
-                    url: this.options.path + 'results',
-                    data: JSON.stringify(request),
-                    dataType: 'json',
-                    contentType: 'application/json',
-                    type: 'POST',
-                    converters: {
-                        'text json': (result) => JSON.parse(result, parseDates),
-                    },
-                    success: (json: IProfiler | string) => {
+
+                fetch(this.options.path + 'results', {
+                    method: 'POST',
+                    body: JSON.stringify(request),
+                    headers: {
+                        'Accept': 'application/json',
+                        'Content-Type': 'application/json'
+                    }
+                })
+                    .then(data => data.text())
```

Comment:
You can use `data.json()` 

https://developer.mozilla.org/en-US/docs/Web/API/Body/json

Reply:
If you look at the next line, I'm not doing _exactly_ a `.json()` (as it's interpreting dates) like on line 390. Is there an overload that takes the custom arg? I didn't find one looking but awesome if there is!

Reply:
Oh I didn't notice that. I would just parse the date after json is parsed. Any reason you need to use the second arg of JSON.parse?

`json.Started = new Date(json.Started);`

Reply:
Yeah, to parse the dates :) No need to allocate twice here (and change types). Note: timings are a tree - traversing that tree to parse all dates would be non-trivial in code and cost...best to do it while deserializing.

Reply:
Makes sense then, don't mind me.